### ISSUE TITLE
Rename to avoid confusion with Leaflet, the mapping library

### ISSUE THREAD

**Issue by mourner** [PotatoProject/Leaflet#75]
Hi @HrX03, can you explain why you picked the name **Leaflet** for an open source project when there's already a widely popular open source project with the same name, a library for interactive maps that existed for 11 years now?

- [34.4k stars on GitHub](https://github.com/Leaflet/Leaflet)
- [29.2k followers on Twitter @LeafletJS](https://twitter.com/LeafletJS)
- [Used by Github, Facebook, the US government and others](https://leafletjs.com)

Please consider renaming the project to avoid confusing users and search engines.

**Comment by dvd-bnc** [comment]
We picked the current name as we ran a contest back then on twitter for the community to choose the name. They proposed some candidates and then proceeded to vote for the one they liked the most and Leaflet (which was a proposal not made by me, you can find the credits on the app) won the overall poll.
The contest was made because we wanted to migrate from the old PotatoNotes name to a more generic one in order to be more appealing.

Now, i knew about the maps project that was named the same way and considered even changing it another time but i avoided because of some specific reasons:
- it didn't infringe copyrights of any kind as leaflet is a term for a common object so it can't be copyrighter either
- it would mean we'd have to scrap the name and kinda "disrespect" the community decision
- i believed that with time people would learn about differentiating the two projects but probably i'm wrong about this

I can still consider changing it but i'd need community approval, a new name and new branding overall

**Comment by mourner** [comment]
@HrX03 thanks for the response! I understand how you want to respect comminity's choice, but they might have not been aware of the existing project, and duplicating the name, opposite to the goal of making it more appealing, now hurts both projects. It's not about copyright of course — but recognition in the open source community matters a lot.

**Comment by dvd-bnc** [comment]
I agree by that, and of course it kinda hurts the project too as searching "leaflet" on google doesn't bring up the app but the obviously more famous library. I'll see what i can figure out and will keep you updated in case
